src/test/java/gov/nih/nci/evs/api/controller/MetadataControllerTests.java

```diff
@@ -1158,6 +1158,17 @@ public void testQualifiersWithoutTypeValue() throws Exception {
     for (Property prop : concept.getProperties()) {
       assertThat(!prop.getType().equals("value"));
     }
+    // Hidden fields are not present
```

Comment:
We should check that testGetProperty(), testGetSynonym(), testGetRole(), are also not returning the hidden fields as well, since the java-examples tests are failing for those because normName is returned